From cc2989f72ba5780cd4d94cc6fad5742c26bcc9c5 Mon Sep 17 00:00:00 2001
Message-Id: <patch-3.7-cc2989f72ba-20220310T134811Z-avarab@gmail.com>
In-Reply-To: <cover-0.7-00000000000-20220310T134811Z-avarab@gmail.com>
References: <cover.1646390152.git.dyroneteng@gmail.com>
	<cover-0.7-00000000000-20220310T134811Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Fri, 4 Mar 2022 12:53:26 +0100
Subject: [RFC/REVIEW 3/7] ls-tree: remove dead labels
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

These have not been used since a preceding "ls-tree: slightly refactor
`show_tree()`" commit, and should be squashed into that.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/ls-tree.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/builtin/ls-tree.c b/builtin/ls-tree.c
index 23ec2f0fe76..f04733a9be1 100644
--- a/builtin/ls-tree.c
+++ b/builtin/ls-tree.c
@@ -18,9 +18,6 @@ static int line_termination = '\n';
 #define LS_RECURSIVE 1
 #define LS_TREE_ONLY (1 << 1)
 #define LS_SHOW_TREES (1 << 2)
-#define LS_NAME_ONLY (1 << 3)
-#define LS_SHOW_SIZE (1 << 4)
-#define LS_OBJECT_ONLY (1 << 5)
 static int abbrev;
 static int ls_options;
 static struct pathspec pathspec;
-- 
2.35.1.1295.g6b025d3e231

